modules/microshift-preparing-to-make-app-rpms.adoc

```diff
@@ -0,0 +1,52 @@
+// Module included in the following assemblies:
+//
+// microshift_running_applications/embed-microshift-apps-on-rhel-edge.adoc
+
+:_content-type: PROCEDURE
+[id="microshift-preparing-to-make-app-rpms_{context}"]
+= Preparing to make application RPMs
+
+To build your own RPMs, choose a tool of your choice, such as the the `rpmbuild` tool, and initialize the RPM build tree in your home directory. The following is an example procedure. As long as your RPMs are accessible to Image Builder, you can use the method you prefer to build the application RPMs.
+
+.Prerequisites
+
+* You have set up a {op-system-ostree-first} {op-system-version} build host that meets the Image Builder system requirements.
+* You have root access to the host.
+
+.Procedure
+
+. Install the `rpmbuild` tool and create the yum repository for it by running the following command:
++
+[source,terminal]
+----
+$ sudo dnf install rpmdevtools rpmlint yum-utils createrepo
+----
+
+. Create the file tree you need to build RPM packages by running the following command:
++
+[source,terminal]
+----
+$ sudo rpmdev-setuptree
```

Comment:
Is this command meant to be executed with elevated privileges (root).  Using root will place the rpmbuild directory will be located in /root/rpmbuild.

Reply:
@DanielFroehlich Could you answer this question, please?

Reply:
Doug is right,  sudo needed here.

Reply:
Are we sure this is right? None of the rest of the commands using that directory are being run as root, so `~/rpmbuild` will be a different directory than `/root/rpmbuild`.

Reply:
Uh, I have to correct my self. I wanted to type sudo is **not** needed here. Sorry Shauna! Please remove the sudo here.